lib/logger/lib/logger/backends/console.ex

```diff
@@ -138,6 +138,12 @@ defmodule Logger.Backends.Console do
   def handle_event({level, _gl, {Logger, msg, ts, md}}, state) do
     %{level: log_level, ref: ref, buffer_size: buffer_size, max_buffer: max_buffer} = state
 
+    {:erl_level, level} = List.keyfind(md, :erl_level, 0, {:erl_level, level})
+    level = case level do
+      :warning -> :warn
+      _ -> level
+    end
```

Comment:
No, let's not convert back. `Logger.warn` is already deprecated, so it is a good opportunity to move to `warning`. :)